clang/lib/Driver/CMakeLists.txt

```diff
@@ -82,6 +87,7 @@ add_clang_library(clangDriver
   ToolChains/ZOS.cpp
   Types.cpp
   XRayArgs.cpp
+  ${FLANG_FILES}
```

Comment:
I wonder if that insertion might be more obvious inserted at the same line as the deletion of Flang.cpp. Minor nit though.

Reply:
Fixed.